### PR TITLE
Bump Xcode version and move build logic to a separated file

### PR DESCRIPTION
**Pull request by rakaramos**
@RocketChat/iOS

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 language: objective-c
-osx_image: xcode8
+osx_image: xcode8.1
```
Comment (rafaelks) [inline .travis.yml]: Now that we have Xcode 8.2, should we update it? :-)

File: Scripts/build.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+xcodebuild clean test -workspace Rocket.Chat.xcworkspace -scheme Rocket.Chat -destination "platform=iOS Simulator,name=iPhone 7" -enableCodeCoverage YES | xcpretty
```
Comment (rafaelks) [inline Scripts/build.sh]: What if we want to add separated process to test iPhone 7+ and SE? Can we just add another script call here?

Comment (rakaramos) [inline Scripts/build.sh]: Yes, we can have all the fun that bash scripts allows us 🎉

File: .travis.yml (hunk 1 above)
Comment (rakaramos) [inline .travis.yml]: If you look at [travis docs](https://docs.travis-ci.com/user/osx-ci-environment/#OS-X-Version), they only support Xcode 8.2 beta... I would say to wait until travis fully adopt it.

Comment (rafaelks) [inline .travis.yml]: Docs are updated now, we can use the new version :-)

Comment (rakaramos) [inline .travis.yml]: :tada:

**Comment by rakaramos** [comment]
@rafaelks updated

**Comment by rafaelks** [comment]
@rakaramos Thank you! 🚀

**Pull request merged**